healthmap/mriids2
# Timespan resets to default when switching between "Data" radio buttons

Fixes : https://github.com/healthmap/mriids2/issues/140

# Test case1:
- pull down the code and run locally or use http://PR-159-mriids.s3-website-us-east-1.amazonaws.com/
- Select "Ebola Outbreak" from Outbreak drop-down menu
       Click on "Timespan" >> 1 year ( Oct 1, 2014 to Oct 1, 2015 )
       Click on Projected cases >> Risk radio button ( Timespan has not changed )
      Then select "Cases" radio button ( Timespan resets to default )
      EXPECTED RESULT- Switching between "Data" radio buttons should not reset the time span to default

# Test case2:
- Do the same above test case by selecting "Covid 19' from Outbreak drop-down menu (The only difference is TimeSpan  range is different for covid outbreak.)

# Test case3:
- Make sure moving between Ebola outbreak and covid outbreak, the dates are back to "initial date rang" for that outbreak.

#Test case 4:
- Run tests using `npm test` and `npm run cypress:cli`. Locally I have already ran the tests and they are succesful.
